src/js/movies.js

```diff
@@ -0,0 +1,19 @@
+/**
+ *
+ * @author Jörgen Lindström
+ */
+//console.log('movies.js loaded');
+
+let moviesArray = [];
+
+export const fetchMovieData = async () => {
+    try {
+        const response = await fetch('../../public/database/movies.json');
+        moviesArray = await response.json(); // Populate the array
+        console.log(moviesArray);
+    } catch (error) {
+        console.error('Error fetching data:', error);
+    }
+};
+
+export { moviesArray };
```

Comment:
I recommend to remove this line and add the export above

Reply:
Done